src/features/swap/index.tsx

```diff
@@ -81,12 +83,23 @@ const SwapWidget = ({ sell }: Params) => {
   const isSwapFeatureEnabled = useIsSwapFeatureEnabled()
   const swapParams = useAppSelector(selectSwapParams)
   const { safeAddress, safeLoading } = useSafeInfo()
-  const [blockedAddress, setBlockedAddress] = useState('')
+  const [recipientAddress, setRecipientAddress] = useState('')
   const wallet = useWallet()
   const { isConsentAccepted, onAccept } = useSwapConsent()
   const feeEnabled = useHasFeature(FEATURES.NATIVE_SWAPS_FEE_ENABLED)
   const useStagingCowServer = useHasFeature(FEATURES.NATIVE_SWAPS_USE_COW_STAGING_SERVER)
 
+  const { data: isSafeAddressBlocked } = useGetIsSanctionedQuery(safeAddress ?? skipToken)
+  const { data: isWalletAddressBlocked } = useGetIsSanctionedQuery(wallet?.address ?? skipToken)
```

Comment:
what I discovered is that skipToken, silences TS for when the value could be undefined, but since we don't want to do API request if the value is an empty string, then we still need to pass {skip: !wallet?.address} as second argument. Otherwise, we are dispatching the event

Reply:
Ah true. Also the recipient is initialized to an empty string so there's the same problem there. Replacing `??` with `||` fixes it, and then we don't need the double skip.

`const { data: isWalletAddressBlocked } = useGetIsSanctionedQuery(wallet?.address || skipToken)`

Reply:
Ok, cool!